.github/workflows/test-ui-e2e-only-desktop.yml

```diff
@@ -32,6 +32,16 @@ on:
         required: false
         type: boolean
         default: false
+      export_to_xray:
+        description: Send tests results to Xray
+        required: false
+        type: boolean
+        default: false
+      test_execution:
+        description: "Test execution be used in Xray"
```

Comment:
can we please name it something like Test Execution ticket ID so it's more clear?

Reply:
done